packages/frontend/core/src/components/blocksuite/block-suite-editor/ai/request.ts

```diff
@@ -83,6 +85,11 @@ async function createSessionMessage({
       })
     );
   }
+  if (retry)
+    return {
+      sessionId,
```

Comment:
I think in @darkskygit's retry impl, the correct way to retry a message is to send a new action with the previous message id